actual-rest-api/openapi.yaml

```
@@ -0,0 +1,156 @@
+openapi: 3.0.0
+x-stoplight:
+  id: i5o5yqr3v45cg
+info:
+  title: University.Schedule.api
+  version: '1.0'
+  license:
+    name: The Unlicense
+    url: 'https://raw.githubusercontent.com/modelix/modelix-samples/main/LICENSE'
+servers:
+  - url: 'http://localhost:8080'
+    description: localDev
+paths:
+  /rooms:
+    get:
+      summary: Provides access to all known rooms
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RoomList'
+      operationId: get-rooms
+  /lectures:
+    get:
+      summary: Provides access to all known lectures
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LectureList'
+      operationId: get-lectures
+  '/rooms/{roomRef}':
+    parameters:
+      - $ref: '#/components/parameters/roomRef'
+    get:
+      summary: Provides access to a single room
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Room'
+      operationId: get-rooms-roomID
+  '/lectures/{lectureRef}':
+    parameters:
+      - $ref: '#/components/parameters/lectureRef'
+    get:
+      summary: Provides access to a single lecture
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Lecture'
+      operationId: get-lectures-lectureRef
+      description: ''
+components:
+  schemas:
+    Room:
+      title: Room
+      x-stoplight:
+        id: sy1if59329olj
+      type: object
+      properties:
+        roomRef:
+          $ref: '#/components/schemas/RoomRef'
+        name:
+          type: string
+        maxPlaces:
+          type: integer
+          minimum: 0
+          format: int32
+        hasRemoteEquipment:
+          type: boolean
+          default: false
+      required:
+        - roomRef
+        - name
+        - maxPlaces
+    Lecture:
+      title: Lecture
+      x-stoplight:
+        id: gfqvd6koxx08s
+      type: object
+      properties:
+        lectureRef:
+          $ref: '#/components/schemas/LectureRef'
+        name:
+          type: string
+        description:
+          type: string
+        maxParticipants:
+          type: integer
+          minimum: 1
+          format: int32
+        room:
+          $ref: '#/components/schemas/RoomRef'
+      required:
+        - lectureRef
+        - name
+        - description
+        - maxParticipants
+        - room
+    RoomRef:
+      title: RoomRef
+      x-stoplight:
+        id: as5v5be7k7j55
+      type: string
```

```suggestion
      type: string
      description: The primary ID of a room. Values are opaque pointers that must not be interpreted.
```

Similarly, for all other Refs?

---

implemented